### PR TITLE
Added mtpaint profile

### PR DESCRIPTION
**Pull request by kortewegdevries**

### CODE REVIEW

File: etc/profile-m-z/mtpaint.profile [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+include disable-shell.inc
+include disable-xdg.inc
+
+include whitelist-runuser-common.inc 
+include whitelist-usr-share-common.inc
+include whitelist-var-common.inc
+
+apparmor
+caps.drop all
+ipc-namespace
```
Comment (rusty-snake) [inline etc/profile-m-z/mtpaint.profile:26]: remove

File: etc/profile-m-z/mtpaint.profile [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+machine-id
+net none
+netfilter
```
Comment (rusty-snake) [inline etc/profile-m-z/mtpaint.profile:29]: remove

Comment (kortewegdevries) [reply etc/profile-m-z/mtpaint.profile:29]: Thanks.

File: etc/profile-m-z/mtpaint.profile (hunk 1 above)
Comment (kortewegdevries) [reply etc/profile-m-z/mtpaint.profile:26]: Why?

Comment (rusty-snake) [reply etc/profile-m-z/mtpaint.profile:26]: It (can) cause graphic/performance issues in X11 depending on distro, xorg.conf, ...

**Comment by rusty-snake** [comment]
Merged, Thanks.

**Pull request merged**